sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/Utils.java

```diff
@@ -552,4 +557,91 @@ public ValueHolder(V v) {
         holder.v = dictionary.remove(key);
         return holder.v != null;
     }
+
```

Comment:
Lets please get clarification on required claims context from OSS

Reply:
Waiting for PR to be approved, and then will start the OSS process. 
Just want to make sure these changes are good and don't posses any hidden faults before starting the OSS process.